dmutils/email/__init__.py

```diff
@@ -0,0 +1,2 @@
+"""Allow accessing send email from email. Backwards compatibility."""
+from dmutils.email.dm_mandrill import send_email
```

Comment:
Fair enough. On a related note, `dm_` in `dm_mandrill` and `dm_notify` seems a bit redundant. It's already in `dmutils`, so I'm not sure if it's adding anything